site/content/docs/concepts/services.ja.md

```diff
@@ -39,8 +39,7 @@ Amazon CloudFront で配信され、S3 でホスティングされた静的 Web
 Application Load Balancer、Network Load Balancer、または両方をトラフィックの入り口として Fargate 上でタスクを実行する ECS サービスです。
 安定したリクエスト量が見込まれる場合、Service から VPC 内のリソースにアクセスする必要がある場合、あるいはより高度な設定の必要がある場合に適した HTTP または TCP サービスの選択肢です。
 
-Application Load Balancer は Environment レベルのリソースであり、Environment 内の全ての Load Balanced Web Service で共有されることに注意しましょう。
-詳細については、[こちら](environments.ja.md#load-balancers-and-dns)を確認してください。対照的に、 Network Load Balancer は Service レベルのリソースであり、 Service 間では共有されません。
+Application Load Balancer は Environment レベルのリソースであり、Environment 内の全ての Load Balanced Web Service で共有されることに注意しましょう。v1.32.0 では、[ワークロード Manifest](../manifest/lb-web-service.ja.md#http-alb) で指定することにより、既存の ALB をサービスレベルでインポートできます。詳細については、[こちら](environments.ja.md#load-balancers-and-dns)を確認してください。対照的に、 Network Load Balancer は Service レベルのリソースであり、 Service 間では共有されません。
```

Comment:
The term "service" and "job" seem to be Copilot term, so could you use "Service レベルでインポートできます"?

Also I understand you translated the original sentence "by specifying it in your [workload manifest]" into "ワークロード Manifest で指定することにより", but I think "Load Balanced Web Service の Manifest で指定することにより" is easier to understand because the referred link is the manifest of Load Balanced Web Service and [a similar sentence](https://github.com/otty246/copilot-cli/blob/ja-docs-periodic-update/site/content/blogs/release-v132.ja.md?plain=1#L53) is translated like that. What do you think?

Reply:
> The term "service" and "job" seem to be Copilot term, so could you use "Service レベルでインポートできます"?

thanks, I will fix it.

> Also I understand you translated the original sentence "by specifying it in your [workload manifest]" into "ワークロード Manifest で指定することにより", but I think "Load Balanced Web Service の Manifest で指定することにより" is easier to understand because the referred link is the manifest of Load Balanced Web Service and [a similar sentence](https://github.com/otty246/copilot-cli/blob/ja-docs-periodic-update/site/content/blogs/release-v132.ja.md?plain=1#L53) is translated like that. What do you think?

I totally agree with you, it would be better "Load Balanced Web Service の Manifest" than "ワークロード Manifest". I will fix it, thanks.